Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptListService.java

```diff
@@ -0,0 +1,78 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.core.IConceptListService;
+import edu.asu.conceptpower.app.core.model.impl.ConceptList;
+import edu.asu.conceptpower.app.core.repository.IConceptListRepository;
+
+/*
+ * 
+ * @author Keerthivasan Krishnamurthy
+ * 
+ * */
+@Service
+public class ConceptListService implements IConceptListService{
+    
+    @Autowired
+    private IConceptListRepository conceptListRepository;
+    
+
+    protected final String LIST_PREFIX = "LIST";
+    
+    @Override
+    public void addConceptList(String name, String description) {
+        ConceptList conceptList = new ConceptList();
+        conceptList.setConceptListName(name);
+        conceptList.setDescription(description);
+        conceptList.setId(generateId(LIST_PREFIX));
+        conceptListRepository.save(conceptList);
+    }
+
+    @Override
+    public void deleteConceptList(String name) {
+        // TODO: Must check whether should we proceed with delete by conceptList name
```

Comment:
No, since we should now have ids for all lists and we're migrating anyways, we should use IDs for everything.